Bottom: 0dbc9a86a20e3dbc96feb1ccbdbd4287f2415691
Top:    d51fb3fb40f977ade683c83f129a7beda9765c51
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2012-12-30 10:23:00 +0000

Change FOUND objects colour from green to purple...

Aim to differentiate the region connected to the search origin from
those which should be (but are not yet) connected to it.


---

diff --git a/src/main.c b/src/main.c
index e41fc55..d80612d 100644
--- a/src/main.c
+++ b/src/main.c
@@ -770,11 +770,11 @@ Color to indicate physical connections. Default: @samp{#00ff00}
 /* %start-doc options "3 Colors"
 @ftable @code
 @item --found-color <string>
-Color to indicate logical connections. Default: @samp{#00ff00}
+Color to indicate logical connections. Default: @samp{#ff00ff}
 @end ftable
 %end-doc
 */
-  COLOR (FoundColor, "#00ff00", "found-color",
+  COLOR (FoundColor, "#ff00ff", "found-color",
 	 "color to indicate logically connected objects"),
 
 /* %start-doc options "3 Colors"
